plugins/linux/l3plugin/descriptor/route.go

```diff
@@ -429,6 +440,17 @@ func (d *RouteDescriptor) Retrieve(correlate []adapter.RouteKVWithMetadata) ([]a
 	return values, nil
 }
 
+// resolves IP address family
+func (d *RouteDescriptor) isIPv6(address string) (isIPv6 bool) {
```

Comment:
I don't see a valid reason to make a method  for a simple function like this. The logging should not be in there anyway.

Reply:
You might have misunderstood me here. I see the point for the helper function itself, I just dont see point making it a METHOD of the descriptor.

Reply:
See below